nltools/data/adjacency.py

```diff
@@ -529,16 +556,17 @@ def write(self, file_name, method='long', **kwargs):
                 'issymmetric': self.issymmetric
             }, compression=kwargs.get('compression', 'blosc'))
         else:
-            if self.is_single_matrix:
-                if method == 'long':
-                    pd.DataFrame(self.data).to_csv(file_name, index=None)
-                elif method == 'square':
-                    pd.DataFrame(self.squareform()).to_csv(file_name, index=None)
-            else:
-                if method == 'long':
-                    pd.DataFrame(self.data).to_csv(file_name, index=None)
-                elif method == 'square':
-                    raise NotImplementedError('Need to decide how we should write out multiple matrices. As separate files?')
+            if (
```

Comment:
This can be simplified to remove the `is_single_matrix` check right, because it doesn't matter for the first condition, all that matters is `if method == 'long'`

Reply:
yeah, thats weird, not sure what I originally had, one of the sourcery.ai refactoring suggestions. I'm now wondering if it actually screwed something up.

Reply:
No I think you're fine. The change I suggested would make it consistent with what we've been doing all along. I think just a weirdo suggestion from sourcery.ai

Reply:
ok, I see what they tried to do. Original version wasn't very clean either.

```
            if self.is_single_matrix:
                if method == 'long':
                    pd.DataFrame(self.data).to_csv(file_name, index=None)
                elif method == 'square':
                    pd.DataFrame(self.squareform()).to_csv(file_name, index=None)
            else:
                if method == 'long':
                    pd.DataFrame(self.data).to_csv(file_name, index=None)
                elif method == 'square':
                    raise NotImplementedError('Need to decide how we should write out multiple matrices. As separate files?')
```

Reply:
fixed